sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/TooManyParametersTest.cs

```diff
@@ -29,10 +31,19 @@ public class TooManyParametersTest
     {
         [TestMethod]
         [TestCategory("Rule")]
-        public void TooManyParameters()
+        public void TooManyParameters_CS()
         {
-            var diagnostic = new TooManyParameters { Maximum = 3 };
-            Verifier.VerifyAnalyzer(@"TestCases\TooManyParameters.cs", diagnostic);
+            Verifier.VerifyAnalyzer(@"TestCases\TooManyParameters.cs",
+                new CSharp.TooManyParameters { Maximum = 3 });
```

Comment:
It's not always applied but I like to have one `_DefaultValue` test and one `_CustomValue` test. This allows avoiding un-seen regression if the default value changes.